packages/voila/index.js

```diff
@@ -0,0 +1,212 @@
+/***************************************************************************
+ * Copyright (c) 2018, Voilà contributors                                   *
+ * Copyright (c) 2018, QuantStack                                           *
+ *                                                                          *
+ * Distributed under the terms of the BSD 3-Clause License.                 *
+ *                                                                          *
+ * The full license is in the file LICENSE, distributed with this software. *
+ ****************************************************************************/
+
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+// Inspired by: https://github.com/jupyterlab/jupyterlab/blob/master/dev_mode/index.js
+
+import './style.css';
+
+import { PageConfig, URLExt } from '@jupyterlab/coreutils';
+
+import { VoilaApp } from '@voila-dashboards/voila';
+
+// Promise.allSettled polyfill, until our supported browsers implement it
```

Comment:
We removed this polyfill upstream in https://github.com/jupyterlab/jupyterlab/pull/11741.

So we can remove it here too if we consider we are targeting the same browsers.